WasteStation/UICode/WSUserInterface.py

```diff
@@ -1,46 +1,67 @@
+import sys
+sys.path.insert(1, '../SensorControls')
 import time
+import runStation
 
 #The value of binID corresponds to each of the three bins in the system.
-#1 = Garbage, 2 = Plastics, 3 = Papers
-def chooseBin(binID):
-	#The following is placeholder code until servo is integrated with UI
-	if(binID == 1):
-		print("Accessing Garbage bin")
-		time.sleep(2)
-		print("Garbage bin is open. Please begin depositing.")
-		time.sleep(2)	
-		return True
-	elif(binID == 2):
-		print("Accessing Plastics bin")
-		time.sleep(2)
-		print("Plastics bin is open. Please begin depositing.")
-		time.sleep(2)
+#1 = Garbage, 2 = Plastics, 3 = Papers, 4 = Compost
+def chooseBin(binID, stationID):
+	#Dictionary containing the channel id and API key of each station
+	stationData = {
+		1 : {'channel_key' : 'MG9FWWZOG8M0PCGK', 'channel_id': '1222563'},
+		2 : {'channel_key' : 'TICC7ZFTEDZOPZ4F', 'channel_id': '1222564'},
+		3 : {'channel_key' : 'A6YJ6ETTQ38WWHBN', 'channel_id': '1222565'}
+	}
+
+	# Value checking for binID and staionID
+	if(not isinstance(binID, int) or not isinstance(stationID, int)):# Checks that binID and stationID are int
+		return False
+	if(binID < 1 or binID > 4): # Only 1 to 4 are valid bin IDs
+		return False
+	if(stationID < 1 or stationID > 3): # Currently there are only 3 stations
+		return False
+
+	result = activateStation(stationData[stationID]['channel_key'], binID)
+	
+	#Bin did not succesfully open
+	if(result == 0):
+		print("Something went wrong. Please try again.")
+		time.sleep(3)
+		return False
+	#Bin successfully opens
+	elif(result == 1):
+		print("Please begin depositing")
```

Comment:
The result is returned after the entire process of activating the station is done. Move this print before _result_ is declared (might look weird in the case of an error or full bin though) or remove it.